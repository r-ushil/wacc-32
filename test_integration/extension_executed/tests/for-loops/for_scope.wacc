#expect this to error
begin
    for (int i = 0; i < 5; i = i + 1) do
        #some scope
        print "hello"
    done ;

    exit i

end