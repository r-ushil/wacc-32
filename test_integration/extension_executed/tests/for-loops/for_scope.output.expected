Compiler exit code: 200
--------- stdout ---------
--------------------------
Exit code: 1
